UnitTests/UnitTests.csproj

```diff
@@ -214,10 +214,6 @@
       <Project>{b1e850ff-dde6-44d5-a830-34250e97a687}</Project>
       <Name>Microsoft.Toolkit.Uwp.Connectivity</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Microsoft.Toolkit.Uwp.Services\Microsoft.Toolkit.Uwp.Services.csproj">
```

Comment:
Why the removal?

Reply:
Twitter implementation moved from Microsoft.Toolkit.Uwp.Services to Microsoft.Toolkit.Services.